velox/core/PlanNode.cpp

```diff
@@ -1315,7 +1316,8 @@ WindowNode::WindowNode(
       outputType_(getWindowOutputType(
           sources_[0]->outputType(),
           windowColumnNames,
-          windowFunctions_)) {
+          windowFunctions_)),
+      binarySearch_(binarySearch) {
```

Comment:
There isn't a need for binarySearch_ field at PlanNode level. If the algorithm shows better results in the benchmark then we can simply make it the default.

Reply:
done